Update readme.md

bkmgit opened the pull request:
Add link to mxnet ruby bindings

I want to contribute to this list and help the maintainers, so:

- [X] I've accepted the terms of the `CC0` License;
- [X] I've added the topic `rubyml` to the contributed repository if appropriate.

bkmgit committed:
Update readme.md

Add link to mxnet ruby bindings

arbox commented on readme.md:
```diff
@@ -156,6 +156,7 @@ programming languages with appropriate bindings for Ruby.
 - [ruby-dnn](https://github.com/unagiootoro/ruby-dnn) - Simple deep learning for Ruby.
 - [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch)
   using [rice](https://github.com/jasonroelofs/rice).
+- [mxnet](https://mxnet.apache.org/) - Ruby bindings for [mxnet](https://github.com/mrkn/mxnet.rb).
```
@bkmgit , could you please exchange the links: Ruby bindings project first, upstream project last.

bkmgit replied:
@arbox Thanks. Done

bkmgit committed:
Update readme.md

Fix as suggested in review to have MXNet Ruby bindings first and library second.

arbox commented on readme.md:
```diff
@@ -154,8 +154,9 @@ programming languages with appropriate bindings for Ruby.
 - [tensorflow](https://github.com/somaticio/tensorflow.rb) - Ruby bindings for [TensorFlow](https://www.tensorflow.org/).
 - [ruby-dnn](https://github.com/unagiootoro/ruby-dnn) - Simple deep learning for Ruby.
-- [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch)
+- [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch).
```
Why do we need a dot here in the middle of a sentence?

bkmgit replied:
All other entries seem to have a full stop, eg lines 155, 154 etc. It seemed that line 157 is missing one.

arbox commented:
@bkmgit ,
thank you for your contribution!
